app-shared-tests/src/main/java/com/appunite/loudius/AbsWalkThroughAppTest.kt

```diff
@@ -62,31 +52,17 @@ abstract class AbsWalkThroughAppTest {
             .respondWithFunction {
                 Instrumentation.ActivityResult(Activity.RESULT_OK, null)
             }
+    }
 
-        composeTestRule.onNodeWithText("Log in").performClick()
-
+    override fun performGitHubLogin() {
         // simulate opening a deeplink
         ActivityScenario.launch<MainActivity>(
             Intent(
                 Intent.ACTION_VIEW,
                 Uri.parse("loudius://callback?code=example_code"),
             ).apply {
-                setPackage(composeTestRule.activity.packageName)
+                setPackage(integrationTestRule.composeTestRule.activity.packageName)
             },
         )
-
-        composeTestRule.waitUntilLoadingDoesNotExist()
```

Comment:
**info**: this part of the logic is in the parent `UniversalWalkThroughAppTest` class